web/src/popover_menus.js

```diff
@@ -468,6 +469,7 @@ export function initialize() {
                 stream_id,
                 topic_name,
                 url,
+                unread_count: unread.num_unread_for_topic(stream_id, topic_name) > 0,
             });
```

Comment:
We should calculate the value of `unread_count` in `popover_menus_data` not the other way around.